File: c#-dotnet-wrapper/CoolBotListConfig.cs
using DSharpPlus;

namespace CoolBotList.NET
{
    class CoolBotListConfig
    {
        public string token;
        public DiscordClient client;
        public int interval;
        public string presence;

        CoolBotListConfig(CoolBotListConfig config)
        {
            this.token = config.token;
            this.client = config.client;
            this.interval = config.interval;
            this.presence = config.presence;
        }
    }
}

File: javascript-typescript-wrapper/src/interfaces/CoolBotListConfig.ts
import { Client } from "discord.js";
import { PresenceResolvable } from "./PresenceResolvable";

export interface CoolBotListConfig {
    /**
     *  The API token.
     */
    token: string;
    /**
     *  The discord client used to make your bot.
     */
    client: Client;
    /**
     *  The interval in ms you want to send data to the API at.
     * The default is 60000.
     */
    interval?: number;
    /**
     * The presence the bot will appear as in the Cool Bot List website.
     */
    presence?: PresenceResolvable;
}
